keylime/tpm/tpm_main.py

```diff
@@ -1200,6 +1270,44 @@ def check_quote(
             hash_alg,
         )
 
+    def check_quote_timing(self, previous_clockinfo, quote):
+        # Sanity check quote clock information
+
+        current_clockinfo = {}
+
+        if previous_clockinfo:
+            retout, success = self._tpm2_printquote(quote, False)
+            if not success:
+                return False, "tpm2_print failed with " + str(retout), current_clockinfo
```

Comment:
As commented IMHO `success` should be dropped. Also in the current code, if `success` is `False`, `retout` is `None`, so the current message will not help.

Reply:
Fair enough, will modify accordingly